lib/widgets/update_post_page.dart

```diff
@@ -0,0 +1,135 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:twitter_clone/theme.dart';
+
+class UpdatePostPage extends StatefulWidget {
+  const UpdatePostPage({Key? key}) : super(key: key);
+
+  @override
+  _UpdatePostPageState createState() => _UpdatePostPageState();
+}
+
+class _UpdatePostPageState extends State<UpdatePostPage> {
+  File? _imageFile;
+
+  Future<void> _pickImage() async {
+    final pickedFile = await ImagePicker().getImage(
+      source: ImageSource.gallery, // Chọn ảnh từ thiết bị
+    );
+
+    if (pickedFile != null) {
+      // Nếu người dùng đã chọn ảnh
```

Comment:
Should write the comments for all your files in English, not combine two languages, this will help our code consistency